Automattic/woocommerce-payments
# Fix empty phone and email value

Fixes 1882-gh-Automattic/woocommerce-payments-server

#### Changes proposed in this Pull Request
Stripe does not accept empty strings for support phone and email. 
We don't have to send empty values for these fields.









#### Testing instructions





* go to `Card readers -> Receipt details` section
* set an empty value for email -> an error should be displayed and the save button disabled
* set an empty value for phone number -> an error should be displayed and the save button disabled

-------------------


**Post merge**
